### PR TITLE
Bigquery Destination: CDK Update

### PR DESCRIPTION
**Pull request by gisripa**
## What
* Upgrade to latest CDK with signature changes.

### CODE REVIEW

**Comment by gisripa** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34468?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* `master`
  * **#34461** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34461?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **#34468** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34468?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **#34467** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34467?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **#34462** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34462?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gisripa and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by edgao — APPROVED** [review]

File: airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java [hunk 1]
```diff
@@ -457,6 +457,11 @@ private TyperDeduper buildTyperDeduper(final BigQuerySqlGenerator sqlGenerator,
 
   }
 
+  @Override
+  public boolean isV2Destination() {
```
Comment (cynthiaxyin) [inline airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java:461]: Thanks! :D

**Review by cynthiaxyin — APPROVED** [review]

**Comment by gisripa** [comment]
### Merge activity

* **Jan 24, 4:37 PM**: @gisripa merged this pull request with [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/34468).

**Pull request merged**